sk/lessons/basics/testing.md

```diff
@@ -128,16 +153,16 @@ defmodule ExampleTest do
   doctest Example
 
   setup_all do
-    {:ok, number: 2}
+    {:ok, recipient: :world}
   end
 
-  test "pravda", state do
-    assert 1 + 1 == state[:number]
+  test "greets", state do
+    assert Example.hello() == state[:recipient]
   end
 end
 ```
 
-## Mockovanie
+## Mocking
```

Comment:
Spomolenina "mockovanie" sa mi tiež nepáči, ale moja skúsenisť hovorí, že ju u nás ľudia bežne používajú. Ale možno som mal len smolu.